eo-maven-plugin/src/main/java/org/eolang/maven/ParseMojo.java

```diff
@@ -159,7 +159,7 @@ private Supplier<Integer> task(final Tojo tojo) {
      * @param tojo Tojo.
      * @return True if the tojo has already been parsed.
      */
-    private boolean hasNotAlreadyParsed(final Tojo tojo) {
+    private boolean isNotParsed(final Tojo tojo) {
```

Comment:
@volodya-lombrozo name of the method contradicts with return description: if `isNotParsed` returns `true` I would expect that an item is not parsed, whereas description says the opposite.

Reply:
@mximp done